configure/os/CONFIG.Common.linux-la64

```diff
@@ -0,0 +1,13 @@
+# CONFIG.Common.linux-la64
+#
+# Definitions for linux-loongarch64 target builds
```

Comment:
> Could you also replace `linux-loongarch64` with `linux-la64` in the comments as well please. Other files below also have similar comments.

Today, I saw an article from the Longson community [如何称呼龙架构？](https://areweloongyet.com/docs/loong-or-loongarch/), and realized that it is necessary to keep the naming conventions of the architecture consistent with those of other open source communities.
Some communities choose to use shorter names like `loong` or `loong64` to refer to the `LoongArch` after the discussion.
Why not call it `la64`?
Because there used to be an architecture called `ia64`, although it has declined, it still exists in the memory of many people. People generally avoid situations where two names have only one difference in `i/l/1` and `O/0` when using Latin letters, which can easily lead to confusion.

|Distro|ARCH identifier|
|------|----------|
|AOSC OS|`loongarch64`|
|Debian|<ul><li>old world：`loongarch64`</li><li>new world：`loong64`</li></ul>|
|Gentoo|`loong`|
|Loong Arch Linux|`loong64`|
|RPM|`loongarch64`|
|Slackware|`loong64`|

|Language|ARCH identifier|
|----|----|
|C#|[`LoongArch64`](https://learn.microsoft.com/zh-cn/dotnet/api/system.runtime.interopservices.architecture?view=net-8.0)|
|Go|`GOARCH=loong64`|
|Rust|`loongarch64-unknown-linux-gnu`|


So I chose to use `loong64` as the name to refer to the LoongArch.